src/client/plugin/usage-info.ts

```diff
@@ -9,7 +9,11 @@ export function getUsageInfo() {
     return "Click the login button to authenticate with GitHub."
   }
 
-  const configBase = "//" + location.host + location.pathname
+  const configBase = (window as any).VERDACCIO_API_URL
+    ? (window as any).VERDACCIO_API_URL
+      .replace(/^https?/, "")
+      .replace(/\/-\/verdaccio\//, "")
```

Comment:
Unfortunately, this doesn't work with the `url_prefix` option. Could you change it up a bit to support that?

DO we still need the `//${location.host}${location.pathname}`, or could we get rid of it? Verdaccio 3 also has the VERDACCIO_API_URL.

Reply:
I kept it in place because there might be custom themes, do you think it matters?

I'll check if there's any place we can reference `url_prefix`.

Reply:
url_prefix is included in VERDACCIO_API_URL

Reply:
> url_prefix is included in VERDACCIO_API_URL, just the replacing doesn't work if it's included

Oh okay. I suspect adding a `$` to the regex will make it work but let me test it out.

Reply:
Actually - what should happen for `url_prefix`? I pushed the fix I mentioned above, but I am not sure if it meets what you have in mind

Reply:
Essentially its just a path prefix that needs to go after the hostname (thats what the location.pathname was for). I think your change should have fixed it. Wil have a look soon.